sass/includes/_alert-message.scss

```diff
@@ -25,13 +25,19 @@
         padding-left: 0.4rem;
         padding-right: 0.4rem;
         margin-right: 0.4rem;
+        font-weight: bold;
+        text-transform: uppercase;
     }
 
     &__description {
         margin-bottom: 0;
         display: inline-block;
     }
 
+    &__description a {
+        color: $color__link-hover;
+    }
+
     &__link {
```

Comment:
Would you be able to remove this `&__link` class, as it is not used if the only way to access the links  is via the `<a>` tag itself

Reply:
No worries.